openshift/imagestreams/jenkins-centos7.json

```diff
@@ -0,0 +1,58 @@
+{
+  "kind": "ImageStream",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "jenkins",
+    "annotations": {
+      "openshift.io/display-name": "Jenkins"
+    }
+  },
+  "spec": {
+    "tags": [
+      {
+        "name": "latest",
+        "annotations": {
+          "openshift.io/display-name": "Jenkins (Latest)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Provides a Jenkins server on CentOS 7. For more information about using this container image, including OpenShift considerations, see https://github.com/openshift/jenkins/blob/master/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Jenkins available on OpenShift, including major versions updates.",
+          "iconClass": "icon-jenkins",
+          "tags": "jenkins"
+        },
+        "from": {
+          "kind": "ImageStreamTag",
+          "name": "2"
+        }
+      },
+      {
+        "name": "1",
+        "annotations": {
+          "openshift.io/display-name": "Jenkins 1.X",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Provides a Jenkins 1.X server on CentOS 7. For more information about using this container image, including OpenShift considerations, see https://github.com/openshift/jenkins/blob/master/README.md.",
+          "iconClass": "icon-jenkins",
+          "tags": "hidden,jenkins",
+          "version": "1.x"
+        },
+        "from": {
+          "kind": "DockerImage",
+          "name": "docker.io/openshift/jenkins-1-centos7:latest"
+        }
+      },
+      {
+        "name": "2",
+        "annotations": {
+          "openshift.io/display-name": "Jenkins 2.X",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Provides a Jenkins v2.x server on CentOS 7. For more information about using this container image, including OpenShift considerations, see https://github.com/openshift/jenkins/blob/master/README.md.",
+          "iconClass": "icon-jenkins",
+          "tags": "jenkins",
+          "version": "2.x"
+        },
+        "from": {
+          "kind": "DockerImage",
+          "name": "docker.io/openshift/jenkins-2-centos7:v3.9"
```

Comment:
So we've got to remember to update the tag here when we cut new release branch, etc. right?

Reply:
yeah.  on the plus side it makes it all be in one place. (sort of. we'll still have to filter it down through library and into origin)